deeposlandia/generator.py

```diff
@@ -152,7 +153,7 @@ def create_generator(dataset, model, datapath, image_size, batch_size, label_con
         Generator of tuples (images, labels), for each input data batch
 
     """
-    if not dataset in ['shapes', 'mapillary', 'aerial']:
+    if not dataset in ['shapes', 'mapillary', 'aerial', 'tanzania', 'open_ai_tanzania']:
```

Comment:
maybe this list should be a global variable

Reply:
You're write, I put it into `deeposlandia/datasets/__init__.py` module. Really useful when associated with the argument parser too!